package.json

```diff
@@ -55,18 +55,25 @@
         "stylelint-formatter-pretty": "^2.1.1",
         "stylelint-no-unsupported-browser-features": "^4.0.0",
         "stylelint-selector-bem-pattern": "^2.1.0",
-        "stylelint-webpack-plugin": "^2.1.1"
+        "stylelint-webpack-plugin": "^2.1.1",
+        "nx": "14.1.9",
```

Comment:
I see that we have nx version freezed. Is that on purpose?

Reply:
i think i can add `^` before it. need to check it first

Reply:
done